peachjam/forms.py

```diff
@@ -169,6 +169,10 @@ def save(self, commit=True):
         if "file" in self.changed_data:
             self.instance.size = None
             self.instance.mimetype = None
+            ext = splitext(self.instance.file.name)[1]
```

Comment:
I think it's useful to keep the uploaded file's filename, because that allows editors to see what was uploaded from their records. This filename is never used externally.

Reply:
ok, we also change it here, in the process upload method, should we be keeping the original file.name?

https://github.com/laws-africa/peachjam/blob/d362a9ece67bb8acebd872c778505eccfcb9c9af/peachjam/forms.py#L47-L51

Reply:
I think we should drop this line, if we want to preserve the original filename?